### ISSUE TITLE
Crash when using custom image

### ISSUE THREAD

**Issue by junninho** [google/restor#59]
Not sure what's going on. When I input the path for the custom image then click go, the app immediately crashes. Report is attached. 
[crash.txt](https://github.com/google/restor/files/2962540/crash.txt)

**Comment by russellhancox** [comment]
Useful bits of the stacktrace:

```
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'NSWindow drag regions should only be invalidated on the Main Thread!'
terminating with uncaught exception of type NSException

15  Restor     0x0000000107598363 -[DownloadImageViewController displayErrorWithFormat:] + 263
16  Restor     0x0000000107597fca __52-[DownloadImageViewController createDownloadSession]_block_invoke.110 + 915
17  Restor     0x000000010759fd63 -[MOLAuthenticatingURLSession URLSession:task:didCompleteWithError:] + 146
```

**Comment by tburgin** [comment]
@jujut1998 There was an error downloading the custom image you provided. Instead of displaying the error to you, Restor was crashing. Fixed in #60. Until we cut a new release you will continue to experience this.

Though, there is something wrong with your url or server that is causing the error to be generated. Fixing that problem would let you use the current version of Restor.
